### DISCUSSION TITLE
Obsidian折腾之路 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#154] [Announcements]
https://blog.goalonez.site/blog/Obsidian%E6%8A%98%E8%85%BE%E4%B9%8B%E8%B7%AF.html 

 Goalonez Blog

**Comment by Sunyin0818** [comment]
推荐插件 `Enveloppe`，可以实现对指定Metadata中的属性筛选，例如public: true，把指定的笔记通过PR的方式推送到Github

**Comment by Goalonez** [comment]
> @Sunyin0818
> 推荐插件 `Enveloppe`，可以实现对指定Metadata中的属性筛选，例如public: true，把指定的笔记通过PR的方式推送到Github

感谢分享
